index.js

```diff
@@ -8,7 +8,20 @@ var TEN_MEBIBYTE = 1024 * 1024 * 10;
 
 module.exports = function (cmd, args, opts) {
 	return new Promise(function (resolve, reject) {
-		var parsed = crossSpawnAsync._parse(cmd, args, opts);
+		var parsed;
+
+		if (opts && opts.__winShell === true) {
```

Comment:
Oh. Stop. This option only needed when we use execa.shell. In execa it shouldn't be.